plugins/easyimage/plugin.js

```diff
@@ -197,10 +212,13 @@
 			},
 
 			onUploaded: function( upload ) {
-				var srcset = CKEDITOR.plugins.easyimage._parseSrcSet( upload.responseData.response );
+				var srcset = CKEDITOR.plugins.easyimage._parseSrcSet( upload.responseData.response ),
+					widget = this;
 
-				this.replaceWith( '<figure class="' + ( editor.config.easyimage_class || '' ) + '"><img src="' +
-					upload.responseData.response[ 'default' ] + '" srcset="' + srcset + '" sizes="100vw"><figcaption></figcaption></figure>' );
+				getNaturalWidth( widget.parts.img, function( width ) {
+					widget.replaceWith( '<figure class="' + ( editor.config.easyimage_class || '' ) + '"><img src="' +
+					upload.responseData.response[ 'default' ] + '" srcset="' + srcset + '" sizes="100vw" width="' + width + '"><figcaption></figcaption></figure>' );
```

Comment:
It should be indented. Generally this HTML string should be a bit more readable.